### PR TITLE
[OT210-15] Adding Organization Entity, repository and script.

### PR DESCRIPTION
**Pull request by RodrigoMdza**
### Motivation and Summary
COMO desarrollador 
QUIERO agregar la entidad Organization
PARA representar en la implementación la estructura de datos

Criterios de aceptación: 
Nombre de tabla: organizations. Los campos son:
name: VARCHAR NOT NULL
image: VARCHAR NOT NULL
address: VARCHAR NULLABLE
phone: INTEGER NULLABLE
email: VARCHAR NOT NULL
welcomeText: TEXT NOT NULL
aboutUsText: TEXT NULLABLE
timestamps y softDelete

### Type of change
- [x] New feature
- [ ] New Tests
- [ ] Bug fix
- [ ] Refactor

### Checklist

- [ ] Traceability between this change and Jira.
- [x] ```mvn clean install``` was run and all tests completed successfully.
- [ ] There are no unused variables and/or imports in the modified classes.
- [ ] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [ ] Slf4j was used for the application log instead ```System.out```.
- [ ] Public methods are documented with ```javadoc```.

### CODE REVIEW

**Comment by RodrigoMdza** [comment]
adding Organization Entity, Repository and Script

File: src/main/resources/db/migration/V1.15__create_table_organization.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+(
+    organization_id  bigint         not null auto_increment,
+    name             varchar(255)   not null,
+    image            varchar(255)   not null,
+    address          varchar(255)   null,
+    phone            integer        null,
+    email            varchar(255)   not null,
+    welcome_text     varchar(255)   not null,
+    about_us_text    varchar(255)   null,
+    slides_id         bigint         not null,
```
Comment (manerajona) [inline src/main/resources/db/migration/V1.15__create_table_organization.sql:13]: Esta relación no va, como muchas slides puede pertenecer a la misma organización la relación va en la tabla slide

File: src/main/resources/db/migration/V1.15__create_table_organization.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    created_at       datetime(6)    not null,
+    is_active        bit            not null,
+    updated_at       datetime(6)    null,
+    primary key (organization_id)
+    foreign key (slides_id) references slides (slides_id)
```
Comment (manerajona) [inline src/main/resources/db/migration/V1.15__create_table_organization.sql:18]: Borrar la FK por favor

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -2,7 +2,7 @@
 spring.datasource.type=com.zaxxer.hikari.HikariDataSource
 spring.datasource.url=jdbc:mysql://localhost:3306/ongdb?createDatabaseIfNotExist=true&useUnicode=true&characterEncoding=UTF-8&serverTimezone=UTC
 spring.datasource.username=root
-#spring.datasource.password=password
```
Comment (manerajona) [inline src/main/resources/application.properties:5]: No subamos este cambio por favor

Comment (RodrigoMdza) [reply src/main/resources/application.properties:5]: Corregido

File: src/main/resources/db/migration/V1.15__create_table_organization.sql (hunk 2 above)
Comment (RodrigoMdza) [reply src/main/resources/db/migration/V1.15__create_table_organization.sql]: Corregido

File: src/main/resources/db/migration/V1.15__create_table_organization.sql (hunk 1 above)
Comment (RodrigoMdza) [reply src/main/resources/db/migration/V1.15__create_table_organization.sql]: Corregido

File: src/main/java/com/alkemy/ong/domain/model/Organization.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    @Column(nullable = false)
+    private String email;
+
+    @Column(name = "welcome_text", nullable = false)
+    private String welcomeText;
+
+    @Column(name = "about_us_text")
+    private String aboutUsText;
+
+    @OneToMany(mappedBy = "organizationId")
```
[the remark this answers is not in the document]
Comment (RodrigoMdza) [reply src/main/java/com/alkemy/ong/domain/model/Organization.java:51]: Corregido

**Pull request merged**